src/simparams.jl

```diff
@@ -42,10 +42,12 @@ struct SimParams
   phone_tracing_params::Union{Nothing, PhoneTracingParams}
 
   infection_modulation_function::Union{Nothing,InfectionModulation}
+  screening_params
```

Comment:
inside a `struct` we need all the types specified

Reply:
ok, i added